extensions/jsonb/deployment/src/main/java/io/quarkus/jsonb/deployment/JsonbProcessor.java

```diff
@@ -47,4 +69,75 @@ public boolean test(BeanInfo bean) {
         }));
     }
 
+    // Generate a JsonbConfigCustomizer bean that registers each serializer / deserializer with JsonbConfig
+    @BuildStep
+    void generateCustomizer(BuildProducer<GeneratedBeanBuildItem> generatedBeans,
+            List<JsonbSerializerBuildItem> serializers,
+            List<JsonbDeserializerBuildItem> deserializers) {
+
+        if (serializers.isEmpty()) {
+            return;
+        }
+
+        final Set<String> customSerializerClasses = new HashSet<>();
+        final Set<String> customDeserializerClasses = new HashSet<>();
+        for (JsonbSerializerBuildItem serializer : serializers) {
+            customSerializerClasses.addAll(serializer.getSerializerClassNames());
+        }
+        for (JsonbDeserializerBuildItem deserializer : deserializers) {
+            customDeserializerClasses.addAll(deserializer.getDeserializerClassNames());
+        }
+        if (customSerializerClasses.isEmpty() && customDeserializerClasses.isEmpty()) {
+            return;
+        }
+
+        ClassOutput classOutput = new ClassOutput() {
+            @Override
+            public void write(String name, byte[] data) {
+                generatedBeans.produce(new GeneratedBeanBuildItem(name, data));
+            }
+        };
+
+        try (ClassCreator classCreator = ClassCreator.builder().classOutput(classOutput)
+                .className("io.quarkus.jsonb.customizer.RegisterSerializersAndDeserializersCustomizer")
+                .interfaces(JsonbConfigCustomizer.class.getName())
```

Comment:
Same note here, why use string and JsonbConfigCustomizer.class.gerName() ?

Reply:
Same answer :)